deployments/charts/kuma-cp/values.yaml

```diff
@@ -0,0 +1,60 @@
+# The default registry and tag to use for all Kuma images
+global:
+  image:
+    registry: "kong-docker-kuma-docker.bintray.io"
+    tag: "0.6.0"
+
+controlPlane:
+  logLevel: "info"
+  mode: "standalone"
+  # zone: ""
+
+  useNodePort: false
+
+  injectorFailurePolicy: Ignore
+
+  service:
+    # name: ""
+    type: ClusterIP
+
+  defaults:
+    skipMeshCreation: true
+
+  #  resources:
+  #     requests:
+  #       cpu: 100m
+  #       memory: 256Mi
```

Comment:
Do we need these?

Reply:
More for documentation purposes. Usually when installing a new chart, I'll copy the chart's values.yaml file and modify from there. We can certainly replace these comments with more structured docs in the README.